hands-on-exercise/2-ignite-cli-adv/8-run-prod.md

```diff
@@ -75,8 +85,8 @@ Make these files publicly downloadable.
 Download them to the node servers, including the validator nodes, and put them in place. You can use a new script `prepare-node.sh` that describes the steps:
 
 ```sh
-cp -v /checkers/build/checkersd-linux-amd64 /usr/local/bin/checkersd
-adduser --home /home/checkersuser --disabled-login --disabled-password checkersuser
+$ cp -v /checkers/build/checkersd-linux-amd64 /usr/local/bin/checkersd
```

Comment:
Are we assuming that the student is using bash? Potentially the student could be using another shell. I personally dont' use the `$` for commands but I'm open for discussion

Reply:
We add the `$` more to show that this is a command, not necessarily bash.
When you use the Copy button on the site, the $ is removed so it does not impair productivity.

Reply:
Ah perfect, that was my next question. If that is the case then we are all good 👍